tests/pytests/unit/test_fileclient.py

```diff
@@ -46,3 +50,45 @@ def test_fileclient_context_manager_closes(temp_salt_minion, temp_salt_master):
 
         assert client._closing
         assert client.channel.close.called
+
+
+@pytest.mark.slow_test
+def test_fileclient_timeout(temp_salt_minion, temp_salt_master):
```

Comment:
```suggestion
def test_fileclient_timeout(temp_salt_minion, temp_salt_master):
```
Why not just `minion_opts` and `master_opts`?